images/prow-tests/Dockerfile

```diff
@@ -197,6 +198,9 @@ RUN go install github.com/google/go-licenses@${GO_LICENSES_VERSION}
 RUN go install sigs.k8s.io/kind@${KIND_VERSION}
 RUN go install github.com/google/go-jsonnet/cmd/jsonnet@${JSONNET_VERSION}
 RUN go install github.com/sigstore/cosign/cmd/cosign@${COSIGN_VERSION}
+RUN go install knative.dev/toolbox/provenance-generator@${TOOLBOX_VERSION}
+RUN go install knative.dev/toolbox/pkg/clustermanager/perf-tests@${TOOLBOX_VERSION}
+RUN go install knative.dev/toolbox/kntest/cmd/kntest@${TOOLBOX_VERSION}
```

Comment:
Isnt this already in the file https://github.com/knative/infra/blob/main/images/prow-tests/Dockerfile#L209-L212 ?

Reply:
:+1:  don't think we need the toolbox_version and can just use latest

Reply:
It is important to pin the version so we can debug errors properly. 

This line is printed for jobs that use this container right at the top and aids troubleshooting.

```
# need to the update url in hack
>> prow-tests image version
Prow test image was built from d6a12e8ca562d4227d3a3de6ed4efe9f7a431012 commit which is viewable at https://github.com/knative/test-infra/tree/d6a12e8ca562d4227d3a3de6ed4efe9f7a431012 
```

Reply:
I'm not sure how the `images/empty.go` works.

Reply:
I like the print the sha version :+1: we don't need to pin it though. the empty.go works by rebuilding this image everytime toolbox is changed. In other words keeps this image pinned with toolbox

Reply:
latest is broken, it seems to pick the 2nd last commit

```
BUILD FAILURE: Build step failure: build step 0 "gcr.io/cloud-builders/docker" failed: step exited with non-zero status: 1
ERROR: (gcloud.builds.submit) build 24917d2b-de29-4a7a-8b65-f32298590fa6 completed with status "FAILURE"
[0m[91mgo: knative.dev/toolbox/kntest/cmd/kntest@latest (in knative.dev/toolbox/kntest@v0.0.0-20230406165419-0318b24a34cf):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
The command '/bin/sh -c go install knative.dev/toolbox/kntest/cmd/kntest@latest' returned a non-zero code: 1
ERROR
ERROR: build step 0 "gcr.io/cloud-builders/docker" failed: step exited with non-zero status: 1
[0m
```